config/imagemaster3000.yml

```diff
@@ -3,7 +3,12 @@ imagemaster3000:
     repository: # Repository from which image definitions will be downloaded
     branch: # Repository branch that will be used
   image-dir: /var/spool/imagemaster3000/ # Directory where to temporarily store images
-  image-list: /var/spool/imagemaster3000/imagemaster3000.list # Name and path of generated image list
+  image-list:
+    path: /var/spool/imagemaster3000/imagemaster3000.list # Name and path of generated image list
+    description: Imagemaster3000 image list # Imagelist description required by cloudkeeper
```

Comment:
Does this work? Don't you need to put the string with spaces into quotes or something?

Reply:
Indeed.